pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/queries/BasicQueryTest.java

```diff
@@ -0,0 +1,32 @@
+package org.apache.pinot.query.runtime.queries;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.pinot.spi.data.FieldSpec;
+
+
+/**
+ * This is a basic test for executiong QueryRunnerTestCases with various different configurations.
+ */
```

Comment:
this is how i envision the test will be written in

Reply:
nice! I like this a lot - it'd be good if we used `@Parameterized` instead of a static block so that we could have individual test results in the TestNG output. Otherwise this is 💯🔥🔥🔥